### DISCUSSION TITLE
rwasm::build failed

### DISCUSSION THREAD

**Discussion by ghdc-ispm** [r-universe-org/help#473] [Q&A]
ERROR: this R is version 4.3.3, package 'mypkg' requires R >= 4.4
Error in wasm_build(pkg, tarball_path, out_dir) : 
  Building wasm binary for package 'mypkg' failed.
Calls: <Anonymous> -> wasm_build
Execution halted

Does this mean the R version on r-universe is 4.3.3 and it failed to build ? How to we get around that ?
Thank you

**Comment by jeroen** [comment]
Can you link to the log please? You can ignore failed builds.
